unsafe/src/main/scala/monocle/unsafe/MapTraversal.scala

```diff
@@ -15,6 +15,18 @@ import scala.collection.immutable.Map
 object MapTraversal {
   implicit def mapEach[K, V]: Each[Map[K, V], V] = fromTraverse[Map[K, ?], V]
 
+  implicit def mapTraversal[K, V]: Traversal[Map[K, V], (K, V)] =
+    new Traversal[Map[K, V], (K, V)] {
+      def modifyF[F[_]: Applicative](f: ((K, V)) => F[(K, V)])(s: Map[K, V]): F[Map[K, V]] =
+        s.toList.traverse { case (k, v) => f(k, v) }.map(kvs => Map(kvs: _*))
+    }
+
+  implicit def mapListIso[K, V]: Iso[Map[K, V], List[(K, V)]] =
+    Iso[Map[K, V], List[(K, V)]](_.toList)(Map.from)
+
+  implicit def mapKVTraversal[K, V]: Traversal[Map[K, V], (K, V)] =
```

Comment:
what's the difference with ` mapTraversal`?

Reply:
Well, they are the same thing. Sorry for not looking into the diff carefully before submitting PR.